setup.py

```diff
@@ -115,10 +115,13 @@ def run(self):
 all_remotes = gs + s3 + azure + ssh + oss + gdrive + hdfs + webhdfs + webdav
 
 terraform = ssh + ["python-terraform>=0.10.1", "jinja2>=2.0.0"]
+dvclive = ["dvclive>=0.2.1"]
```

Comment:
Is there a reason to not add it directly into `test_requirements.txt`?

Reply:
Not really. I just wasn't sure if `test_requirements.txt` was reserved to "strict" test requirements given that `dvclive` depdency is optional and tests would be skipped

Reply:
Let's put it into `test_requirements.txt` directly then.